spec/configuration_spec.rb

```diff
@@ -57,4 +57,18 @@
 
     expect(ISO3166.configuration.loaded_locales.size).to eq 92
   end
+
+  context 'lazy load default locales' do
+    it 'does not load default locales during initialization' do
+      expect(I18n).to receive(:available_locales).never
+
+      ISO3166::Configuration.new
+    end
+
+    it 'loads default locales when calling #locales for the first time' do
+      expect(I18n).to receive(:available_locales).once.and_return([:test])
+
+      expect(ISO3166::Configuration.new.locales).to eq([:test])
+    end
+  end
```

Comment:
I don't know if those tests are needed.
They helped me to make the changes, but they might not provide long-term value.

Happy to remove them.